src/controllers/profileController.js

```diff
@@ -62,6 +62,25 @@ class ProfileManager {
       });
     }
   }
+
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} all users are returned
+   */
+  static async getAllProfiles(req, res) {
```

Comment:
change the name to be more descriptive to your task; you are returning a list of users with their profiles